v_4_0_0/master_template.go

```diff
@@ -306,6 +306,18 @@ systemd:
       ExecStartPost=/usr/bin/systemctl restart k8s-kubelet.service
       [Install]
       WantedBy=multi-user.target
+  
+  - name: debug-tools.service
+    enabled: true
+    contents: |
+      [Unit]
+      Description=Calicoctl and crictl tools
```

Comment:
Should it be following?

```suggestion
      Description=Install calicoctl and crictl tools
```